muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/frontpage/FrontPageJsPluginMessages_en.properties

```diff
@@ -94,7 +92,10 @@ plugin.evaluation.evaluation = Evaluation
 plugin.announcer.announcer = Announcer
 plugin.records.records = Transcript of Records
 
-plugin.profileBadge.links.personalInfo = Personal Info
+plugin.profileBadge.links.personalInfo = My Information
```

Comment:
Finnish text string is "Omat tiedot" so what would be the correct one? Those are just synonyms for same thing, it could be "Personal Info/Omat tiedot" in the future or "My Information/Minun tietoni" or something in between.